docs/mysql.md

```diff
@@ -0,0 +1,60 @@
+# Using MySQL
+
+The Docker configuration in this repository defaults to using PostgreSQL.
+If you prefer to work with MySQL, some adjustments need to be made.
+
+First, install the doctrine/orm pack as described: `make composer c='req symfony/orm-pack`
+
+## Docker configuration
+Change the database image to use mysql instead of postgresql in `compose.yaml`:
```

Comment:
```suggestion
Change the database image to use MySQL instead of PostgreSQL in `compose.yaml`:
```

Reply:
yes, my intention was to mention the image names. I think that we should use DBMS Name instead.